docs/cheatsheet.rst

```diff
@@ -25,10 +26,23 @@ Global Variables
   tuple. Performs a full type-check, ensuring the types match the function signature. Result equals ``abi.encodeWithSelector(functionPointer.selector, (...))``
 - ``abi.encodeWithSignature(string memory signature, ...) returns (bytes memory)``: Equivalent
   to ``abi.encodeWithSelector(bytes4(keccak256(bytes(signature))), ...)``
+  
+Members of bytes
```

Comment:
I'd group things for `string` and `bytes` in a single section.

Reply:
Not sure, What should I name it so I just name it " Members of bytes and string "

Reply:
Yeah. "Members of `bytes` and `string`" sounds just fine.